src/views/requestAccess.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from "react";
+import { useHistory } from 'react-router-dom';
+import { ACCESS_REQUEST_DATA } from '../components/DashboardModule/data';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+
```

Comment:
Can we go through this file and make sure lines and white space are formatted correctly?